src/engine/execute.js

```diff
@@ -112,8 +112,8 @@ const handlePromise = (primitiveReportedValue, sequencer, thread, blockCached, l
     // Promise handlers
     primitiveReportedValue.then(resolvedValue => {
         handleReport(resolvedValue, sequencer, thread, blockCached, lastOperation);
-        // If its a command block.
-        if (lastOperation && typeof resolvedValue === 'undefined') {
+        // If its a command block or a top level reporter in a stackClick.
```

Comment:
May as well fix this `its` (-> `it's`) typo while you're changing this line of code?